src/components/portal/portal.js

```diff
@@ -1,35 +1,63 @@
 import React from 'react';
-import ReactPortal from 'react-portal';
 import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
+import Browser from '../../utils/helpers/browser';
 
 class Portal extends React.Component {
-  static propTypes = {
-    /**
-     * Determine if the portal is visible or not
-     *
-     * @property open
-     * @type {Boolean}
-    */
-    open: PropTypes.bool,
 
+  static propTypes = {
     /**
      * The content of the portal.
      *
      * @property children
      * @type {Node}
      */
-    children: PropTypes.node
+    children: PropTypes.node,
+    /**
+     * The DomNode 
+     *
+     * @property node
+     * @type {Any}
+     */
+    node: PropTypes.any
+  }
+
+  componentWillUnmount() {
+    if (this.defaultNode) {
+      Browser.getDocument().body.removeChild(this.defaultNode);
+    }
+    this.defaultNode = null;
+  }
+
+  getDefaultDiv() {
+    
+    if (!this.props.node && !this.defaultNode && Browser.isDomAvailable()) {
+      this.defaultNode = Browser.getDocument().createElement('div');
+      this.defaultNode.classList.add('portal');
+      document.body.appendChild(this.defaultNode);
```

Comment:
retrieve document from `Browser.getDocument` and use it here too.

Reply:
Opps. Done